WcaOnRails/spec/requests/registrations_spec.rb

```diff
@@ -546,7 +546,7 @@
             payment_method_id: pm.id,
             amount: registration.outstanding_entry_fees.cents,
           }
-          stripe_charge_id = registration.registration_payments.first.stripe_charge_id
+          stripe_charge_id = registration.reload.registration_payments.first.stripe_charge_id
```

Comment:
@jonatanklosko this was causing (quite logically) a failure in the specs, but what I don't understand is how it was working before...
I'm not sure what's involved in rails figuring out what should be reloaded or not, but clearly the reload has to happen for the expects below to work, so I fixed it.

Reply:
That actually makes sense! The test uses `outstanding_entry_fees` above, which in turn calls `paid_entry_fees` for the registration: https://github.com/thewca/worldcubeassociation.org/blob/94af2e62e26c9e83a05bc358611fb9c185c5f058/WcaOnRails/app/models/registration.rb#L119-L121

Before it would just perform a `SUM` SQL query every time, but with the new implementation it loads registration payments, so in this line `registration.registration_payments` returns the cached payments and we need to reload them =)

Reply:
Ah nice catch :)